modules/secure-logging/secure-logging.c

```diff
@@ -75,14 +77,13 @@ tf_slog_prepare(LogTemplateFunction *self, gpointer s, LogTemplate *parent, gint
 {
   // Get key filename and store in internal state
   // generate initial BigMAC file
-  TFSlogState *state = (TFSlogState *) s;
+  TFSlogState *state = (TFSlogState *)s;
 
   gchar *keypathbuffer = NULL;
   gchar *macpathbuffer = NULL;
   GOptionContext *ctx;
   GOptionGroup *grp;
-
-  if ((mlock(state->key, KEY_LENGTH)!=0)||(mlock(state->bigMAC, CMAC_LENGTH)!=0))
+  if ((mlock(state->key, KEY_LENGTH)!=0)||(mlock(state->aggMAC, CMAC_LENGTH)!=0))
```

Comment:
i do not know why the style check does not complain about this, but please add spaces for readability